src/Core/ServiceWrapper/winsw.csproj

```diff
@@ -1,4 +1,4 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
BTW which IDE do you use?

Reply:
I'm using the latest Visual Studio 2019.